evalml/tests/conftest.py

```diff
@@ -860,11 +860,11 @@ def __init__(
 
 
 @pytest.fixture
-def dummy_ts_binary_linear_classifier_pipeline_class():
-    log_reg_classifier = LogisticRegressionClassifier
+def dummy_ts_binary_tree_classifier_pipeline_class():
```

Comment:
Why did we need to change the estimator used here?

Reply:
The linear regressor doesn't support datetime columns and throws `TypeError: The DType <class 'numpy.dtype[datetime64]'> could not be promoted by <class 'numpy.dtype[float64]'>.`

With `DecisionTreeClassifier`, we can support datetime columns while still choosing an estimator supported with minimum requirements.

Reply:
Thanks for explaining!